test/unit/MerkleDrop.spec.js

```diff
@@ -1,7 +1,183 @@
 const { expect } = require("chai");
-const { utils } = require("ethers");
-const { setupFixture, setupInitialState } = require("./utils/prepareState");
+const { utils, BigNumber } = require("ethers");
+const BN = require("bn.js");
 const { ethers } = require("hardhat");
+const {
+  getTranche,
+  createTreeWithAccounts,
+  getAccountBalanceProof,
+} = require("../../tasks/utils/merkle");
+
+const rawAllocations = [
+  "1056.39331099660171709",
+  "870.94684945018301126",
+  "226.48095676324248635",
+  "7.46007341986023537",
+  "9.64320061541387227",
+  "45.45852965164066046",
+  "2.58531507323313161",
+  "105.44076288303354762",
+];
+
+const getParsedAllocations = (addresses, rawAllocations) =>
```

Comment:
Learning much from your PR. So going to ask some questions to check if I really get what is happening here. This is to top up the balances of the accounts with Ether in an easy way I guess?

Reply:
This function matches the amounts of v2 tokens that we want to distribute to the test addresses that are generated automatically by hardhat, so that we can check in the tests, if a certain test address was in fact able to claim its allocated v2 amount. What comes out of this function is something like this:
```
{
  <testAddress1>: <allocation1>,
  <testAddress2>: <allocation2>,
  ...
}
```
I renamed the function to make this clearer.

Reply:
Thanks, it's clear now